clustered-indexes.md

```diff
@@ -0,0 +1,188 @@
+|     └─TableFullScan_5     | 2.00    | cop[tikv] | table:does_not_cluster_by_default | keep order:false, stats:pseudo                 |
++---------------------------+---------+-----------+-----------------------------------+------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+# Changes since TiDB 5.0
+
+Since v5.0, TiDB supports clustering by any `PRIMARY KEY`. The following `EXPLAIN` output shows the previous example with clustered indexes enabled:
+
+```sql
+SET tidb_enable_clustered_index = 1;
+CREATE TABLE will_now_cluster (
+ guid CHAR(32) NOT NULL PRIMARY KEY,
+ b CHAR(100),
+ INDEX(b)
+);
+
+INSERT INTO will_now_cluster VALUES (1, 'aaa'), (2, 'bbb');
+INSERT INTO will_now_cluster VALUES ('02dd050a978756da0aff6b1d1d7c8aef', 'aaa'), ('35bfbc09cb3c93d8ef032642521ac042', 'bbb');
+
+EXPLAIN SELECT * FROM will_now_cluster WHERE guid = '02dd050a978756da0aff6b1d1d7c8aef';
+EXPLAIN SELECT guid FROM will_now_cluster WHERE b = 'aaaa';
+```
+
+```sql
+Query OK, 0 rows affected (0.00 sec)
+
+Query OK, 0 rows affected (0.11 sec)
+
+Query OK, 2 rows affected (0.02 sec)
+Records: 2  Duplicates: 0  Warnings: 0
+
+Query OK, 2 rows affected (0.01 sec)
+Records: 2  Duplicates: 0  Warnings: 0
+
++-------------+---------+------+-------------------------------------------------------+---------------+
+| id          | estRows | task | access object                                         | operator info |
++-------------+---------+------+-------------------------------------------------------+---------------+
+| Point_Get_1 | 1.00    | root | table:will_now_cluster, clustered index:PRIMARY(guid) |               |
++-------------+---------+------+-------------------------------------------------------+---------------+
+1 row in set (0.00 sec)
+
++--------------------------+---------+-----------+------------------------------------+-------------------------------------------------------+
+| id                       | estRows | task      | access object                      | operator info                                         |
++--------------------------+---------+-----------+------------------------------------+-------------------------------------------------------+
+| Projection_4             | 10.00   | root      |                                    | test.will_now_cluster.guid                            |
+| └─IndexReader_6          | 10.00   | root      |                                    | index:IndexRangeScan_5                                |
+|   └─IndexRangeScan_5     | 10.00   | cop[tikv] | table:will_now_cluster, index:b(b) | range:["aaaa","aaaa"], keep order:false, stats:pseudo |
++--------------------------+---------+-----------+------------------------------------+-------------------------------------------------------+
+3 rows in set (0.00 sec)
+```
+
+Clustering by a composite `PRIMARY KEY` is also supported:
```

Comment:
```suggestion
Clustered index by a composite `PRIMARY KEY` is also supported:
```

Reply:
I have used clustering earlier in the document already. It refers to "organizing data" by a composite key.